Setup-Your-Mac-via-Dialog.bash

```diff
@@ -235,18 +235,42 @@ updateScriptLog "PRE-FLIGHT CHECK: Current Logged-in User ID: ${loggedInUserID}"
 # Pre-flight Check: Temporarily disable `jamf` binary check-in (thanks, @mactroll and @cube!)
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
+function toggleJamfLaunchDaemon() {
+
+jamflaunchDaemon="/Library/LaunchDaemons/com.jamfsoftware.task.1.plist"
+
 if [[ "${debugMode}" == "true" ]] || [[ "${debugMode}" == "verbose" ]] ; then
-    updateScriptLog "PRE-FLIGHT CHECK: DEBUG MODE: Normally, 'jamf' binary check-in would be temporarily disabled"
+	if [[ $(/bin/launchctl list | grep com.jamfsoftware.task.E) ]]; then
+		updateScriptLog "PRE-FLIGHT CHECK: DEBUG MODE: Normally, 'jamf' binary check-in would be temporarily disabled"
+	else
+		updateScriptLog "QUIT SCRIPT: DEBUG MODE: Normally, 'jamf' binary check-in would be re-enabled"
+	fi
 else
-    updateScriptLog "PRE-FLIGHT CHECK: Temporarily disable 'jamf' binary check-in"
-    jamflaunchDaemon="/Library/LaunchDaemons/com.jamfsoftware.task.1.plist"
-    while [[ ! -f "${jamflaunchDaemon}" ]] ; do
```

Comment:
[Use grep -q instead of comparing output with [ -n .. ].](https://www.shellcheck.net/wiki/SC2143)